scala-package/examples/pom.xml

```diff
@@ -119,6 +119,14 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
+        <executions>
```

Comment:
Would  the child poms not inherit this execution stage from the parent pom file ?